### ISSUE TITLE
Macro inlining: doesn't work in module located in non-root file

### ISSUE THREAD

**Issue by W95Psp** [hacspec/hax#102]
This makes `merlin` not to extract (related to #28)

The bug is that only the surface AST of the root `rs` file is parsed when collecting macro invocations.

**Status:** I've added a `wontfix` label on this issue, since I think we are moving toward dropping specially handled macros

**Comment by W95Psp** [comment]
Waiting on #145
